tutorial/tutorial.md

```diff
@@ -0,0 +1,333 @@
+## Tutorial on how to setup a local installation of RHOC
+
+This tutorial will show how to create a local installation of RHOC and deploy a simple Slack connector.
+
+We'll use the `kas-fleet-manager` repository, ignoring `cos-fleet-manager` because most of the code it's in this repository
+and it's easier to setup a system by using a single control plane repository. 
+You can consider `cos-fleet-manager` the midstream repository of the control plane project.
+
+```cd kas-fleet-manager```
+
+## Setup the DB
+
+`make db/teardown`
+
+This will fail if the DB doesn't exist
+
+`make db/setup`
+
+We can check a docker container called `kas-fleet-manager-db` has been created with
+
+```
+docker container ls
+make db/login
+```
+
+This will login inside the database. With `\d` you can see there are no tables.
+We should run the migration to populate the DB with the RHOC tables
+
+To run the RHOC migrations we can use the `internal/connector/test/main/main.go` main instead of the "normal" `cmd/main.go`, which is used for `kas`. 
+This `main.go` let us run a RHOC control plane inside the `kas-fleet-manager` repository.
+
+To run the migrations: 
+
+`go run internal/connector/test/main/main.go migrate`
+
+or use the debug action from GoLand specifying `migrate` as a command line argument
+
+We can check the tables are created accordingly with
+
+```
+make db/login
+\d
+```
+
+To start then the control plane either run it from the local action from GoLand specifying `serve` as a command line argument or use
+
+`go run internal/connector/test/main/main.go serve --sso-provider-type=redhat_sso`
```

Comment:
Not sure if `make sso/setup` or `make ocm/setup` or `make keycload/setup` are needed?

@vaibhavjainwiz was having problems with starting things and missing files.

Not sure if 1, 2 or 3 of these setups are needed?

Reply:
with local keycloak, there are instructions on how to use those targets